src/main/java/uk/gov/hmcts/reform/et/syaapi/notification/NotificationsProperties.java

```diff
@@ -30,7 +30,7 @@ public class NotificationsProperties {
     @NotEmpty
     private String submitCaseEmailTemplateId;
 
-    @Value("https://www.gov.uk/log-in-register-hmrc-online-services")
+    @Value("https://et-sya.perftest.platform.hmcts.net/citizen-hub/")
```

Comment:
I think value needs to come from environment variables

Reply:
Updated as an environment variable.